packages/compass/src/app/components/home.tsx

```diff
@@ -318,6 +362,11 @@ function Home({
                 getAutoConnectInfo={
                   hasDisconnectedAtLeastOnce ? undefined : getAutoConnectInfo
                 }
+                useConnectionImportExportModalRenderer={
+                  useConnectionImportExportModalRenderer
+                }
+                renderLegacyConnectionModal={() => <LegacyConnectionsModal />}
```

Comment:
This modal doesn't take any props and pulls all the required data from the context, right? Doesn't look to me that we need to render it through the CompassConnections component here, or am I missing something?

Reply:
That's right. I moved this along with Import Export connection modals to Home